### PR TITLE
Kapakrit

### PR DESCRIPTION
**Pull request by sainho93**
Add new geo XML file parser and writer for #148

### CODE REVIEW

File: src/mainWindow.cpp [hunk 1]
```diff
@@ -513,6 +514,7 @@ void MWindow::openFileXML()
 
     if(!error_goal.isEmpty() or !error_routing.isEmpty() or !error_source.isEmpty() or !error_traffic.isEmpty())
     {
+        // Show error information when some files are lacked
```
Comment (zeroset) [inline src/mainWindow.cpp]:
```suggestion
        // Show error information when some files are locked
```

Comment (sainho93) [reply src/mainWindow.cpp]: Here I really mean that some files are lacked, it means not all files in the folder.

File: src/mainWindow.cpp [hunk 2]
```diff
@@ -513,6 +514,7 @@ void MWindow::openFileXML()
 
     if(!error_goal.isEmpty() or !error_routing.isEmpty() or !error_source.isEmpty() or !error_traffic.isEmpty())
     {
+        // Show error information when some files are locked
```
Comment (zeroset) [inline src/mainWindow.cpp]:
```suggestion
        // Show error information when some files are missing
```

Ok then batter missing?

Comment (sainho93) [reply src/mainWindow.cpp]: Agreed!

**Pull request merged**